src/shimoku_api_python/api/plot_api.py

```diff
@@ -7672,6 +7672,157 @@ def line_with_confidence_area(self,
             }
         )
 
+    @logging_before_and_after(logging_level=logger.info)
+    def tornado(
+            self,
+            menupath: str,
+            order: int,
+            df_neg: pd.DataFrame, df_pos: pd.DataFrame,
+            cols_size=12,
+            rows_size=5,
+            neg_col_val="value", pos_col_val="value",
+            neg_col_name="name", pos_col_name="name",
+            options_mod={}, neg_dataops_mod={},
+            pos_dataops_mod={}, seriesopt_mod={},
```

Comment:
We need more parameters for all the options that the module provides: tabs_index, modal_name, bentobox, etc.